R/zchunk_L2322.Fert.R

```diff
@@ -76,272 +76,447 @@ module_energy_L2322.Fert_DISABLED <- function(command, ...) {
+      add_precursors("energy/A322.subsector_shrwt", 'common/GCAM_region_names') ->
       L2322.SubsectorShrwt_Fert
 
-    tibble() %>%
-      add_title("descriptive title of data") %>%
-      add_units("units") %>%
-      add_comments("comments describing how data generated") %>%
-      add_comments("can be multiple lines") %>%
+    L2322.SubsectorShrwtFllt_Fert %>%
+      add_title("Subsector shareweights of fertilizer") %>%
+      add_units("Unitless") %>%
+      add_comments("Expanded Subsector shareweights of fertilizer to all GCAM regions") %>%
       add_legacy_name("L2322.SubsectorShrwtFllt_Fert") %>%
-      add_precursors("precursor1", "precursor2", "etc") %>%
-      # typical flags, but there are others--see `constants.R`
-      add_flags(FLAG_LONG_YEAR_FORM, FLAG_NO_XYEAR) ->
+      add_precursors("energy/A322.subsector_shrwt", "common/GCAM_region_names") ->
       L2322.SubsectorShrwtFllt_Fert
 
-    tibble() %>%
-      add_title("descriptive title of data") %>%
-      add_units("units") %>%
-      add_comments("comments describing how data generated") %>%
-      add_comments("can be multiple lines") %>%
+    L2322.SubsectorInterp_Fert %>%
+      add_title("Subsector shareweight interpolation of fertilizer sector") %>%
+      add_units("NA") %>%
+      add_comments("Expanded subsector shareweight interpolation of fertilizer sector to all GCAM regions") %>%
       add_legacy_name("L2322.SubsectorInterp_Fert") %>%
-      add_precursors("precursor1", "precursor2", "etc") %>%
-      # typical flags, but there are others--see `constants.R`
-      add_flags(FLAG_LONG_YEAR_FORM, FLAG_NO_XYEAR) ->
+      add_precursors("energy/A322.subsector_interp", "common/GCAM_region_names") ->
       L2322.SubsectorInterp_Fert
 
-    tibble() %>%
-      add_title("descriptive title of data") %>%
-      add_units("units") %>%
-      add_comments("comments describing how data generated") %>%
-      add_comments("can be multiple lines") %>%
+    L2322.SubsectorInterpTo_Fert %>%
+      add_title("Subsector shareweight interpolation of fertilizer sector") %>%
+      add_units("NA") %>%
+      add_comments("Created as empty table") %>%
       add_legacy_name("L2322.SubsectorInterpTo_Fert") %>%
-      add_precursors("precursor1", "precursor2", "etc") %>%
-      # typical flags, but there are others--see `constants.R`
-      add_flags(FLAG_LONG_YEAR_FORM, FLAG_NO_XYEAR) ->
+      add_precursors("energy/A322.subsector_interp", "common/GCAM_region_names") ->
       L2322.SubsectorInterpTo_Fert
 
-    tibble() %>%
-      add_title("descriptive title of data") %>%
-      add_units("units") %>%
-      add_comments("comments describing how data generated") %>%
-      add_comments("can be multiple lines") %>%
+    L2322.StubTech_Fert %>%
+      add_title("Identification of stub technologies of fertilizer sector") %>%
+      add_units("NA") %>%
+      add_comments("Expanded identification of stub technologies of fertilizer sector to all GCAM regions") %>%
```

Comment:
What does "identification" mean?

Reply:
Copied from old chunk. The table contains Stub-technology (coal, coal CCS, and etc.) for fertilizer sector. Any suggestion to replace the word `identification`?

Reply:
What about just what you said: "Stub-technology (coal, coal CCS, and etc.) for fertilizer sector"?

Reply:
Sounds good. Thanks.